homeassistant/components/squeezebox/quality_scale.yaml

```diff
@@ -0,0 +1,64 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration only has entity_actions, which are setup in the entity async_setup_entry.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
`test_form_cannot_connect` -> test until the end

Reply:
Same for `test_discovery` and `test_discovery_no_uuid` and `test_dhcp_discovery` and `test_dhcp_discovery_no_server_found`

Reply:
Done, so the conversation can be resolved?